src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -114,7 +105,9 @@ public JsonSerializerOptions(JsonSerializerOptions options)
             _includeFields = options._includeFields;
             _propertyNameCaseInsensitive = options._propertyNameCaseInsensitive;
             _writeIndented = options._writeIndented;
-
+            // Preserve backward compatibility with .NET 6
```

Comment:
TODO: Determine semantics of copy constructor here before merging. We might want to consider introducing a breaking change, but this will imply a one-to-many relationship between `JsonSerializerContext` and `JsonSerializerOptions`.

Reply:
I'll suggest to to this change separately so it's easier to track fixes or product changes like this